app/models/user.rb

```diff
@@ -5,10 +5,15 @@ class User < ApplicationRecord
   include Roles
   include ByOrganizationScope
   include DateHelper
+  include ActiveModel::Dirty
```

Comment:
Oh that's good to know! I had included it because, from the way I read the docs, I thought it suggested it wasn't already available. Would you recommended double checking if a module is already included by doing something like: 

```ruby
ApplicationRecord.included_modules.include?(ActiveModel::Dirty)
```